kernel/kernel-impl/src/main/webapp/WEB-INF/components.xml

```diff
@@ -11,7 +11,7 @@
 
      <context:annotation-config />
      <!-- We need to scan these to pick up the bullhorn handlers for MessagingServiceImpl -->
-     <context:component-scan base-package="org.sakaiproject.messaging.api.testhandler,org.sakaiproject.announcement.api, org.sakaiproject.profile2.model, org.sakaiproject.commons.api, org.sakaiproject.lessonbuildertool.api" />
+     <context:component-scan base-package="org.sakaiproject.messaging.api.testhandler, org.sakaiproject.profile2.model, org.sakaiproject.commons.api, org.sakaiproject.lessonbuildertool.api" />
```

Comment:
With the registration now incumbent upon the handlers themselves, I think this component-scan is unnecessary (and inappropriate). If we get to a point where tool-owned components are in their own tier (context), this kind of scan would always come up empty. It only kind-of works now because everything goes into one context.